### ISSUE TITLE
Test cases with blank space names does not run in individual mode 

### ISSUE THREAD

**Issue by halfow** [Bochlin/vunit-test-explorer#7]
Tests with blank spaces will not run in "individual mode" they exit with `"No tests were run!"`

Example `if run("Test A") then` 

Today:
`python run.py --exit-0 -v my_project.tb_test.Test A` 

Sugested: (add quotes)
`python run.py --exit-0 -v "my_project.tb_test.Test A"`

Note: added `-v` in CLI options  for debugging :)

**Comment by halfow** [comment]
Patch to this in `v0.3.0` broke run all

Output run all in `v0.3.0`:
````
Running VUnit: python <PATH>/run.py --no-color --exit-0 -v ""
Re-compile not needed

Running 0 tests

No tests were run!

Finished with exit code 0
````

Running form CLI adding wildcard `*`  solves the problem. This would be a hack to get it to work for both cases. Conditional statement for run all respective run single evaluation would remove possible failure modes and should thus be considered. 

`python <PATH>/run.py --no-color --exit-0 -v "*<TESTCASE INJECTION POINT>"`


Thanks for your work! I will try to learn some TypeScript so I can be more involved

### Edit: 
looking through the diff with 0.2.0 `src/vunit.ts:152` stood out as a possible candidate 0.2.0@`testNames.push('*');` &  0.3.0@`testNames.push('');`

**Comment by Bochlin** [comment]
Using * caused problems on Linux which is why it was removed. I have pushed another fix which does not print the arguments when empty.

This projects lacks proper tests which is something that I need to get fixed, but time is limited...

**Comment by halfow** [comment]
I confirmed the fix of `v0.3.2` works under windows as well :)
